src/applications/accredited-representative-portal/components/POARequestDetails/POARequestDetails.jsx

```diff
@@ -0,0 +1,174 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useParams } from 'react-router-dom-v5-compat';
+import usePOARequests from '../../hooks/usePOARequests';
+import { formatDate } from '../POARequestsTable/POARequestsTable';
+
+const POARequestDetails = () => {
+  const { poaRequests } = usePOARequests();
+  const { id } = useParams();
+
+  const details =
+    poaRequests &&
+    poaRequests
+      .filter(request => request.id === Number(id))
+      .map(request => request.attributes);
+
+  const checkAuthorizations = (
+    isTreatmentDisclosureAuthorized,
+    isAddressChangingAuthorized,
+    status,
+  ) => {
+    const authorizations = [];
+    if (isTreatmentDisclosureAuthorized === status) {
+      authorizations.push('Health');
+    }
+
+    if (isAddressChangingAuthorized === status) {
+      authorizations.push('Address');
+    }
+    return authorizations.length > 0 ? authorizations.join(', ') : 'None';
+  };
```

Comment:
Let's move this helper function to the top level outside the React functional component function because it is a helper function of only its arguments.